.github/workflows/release-please.yml

```diff
@@ -24,10 +24,9 @@ jobs:
         run: |
           npm ci
 
-      - uses: google-github-actions/release-please-action@v3
+      - uses: google-github-actions/release-please-action@v4
         id: release
         with:
-          command: manifest
           token: ${{secrets.RELEASE_PR_TOKEN}}
           default-branch: main
```

Comment:
nit: Is `default-branch` a thing with this action? https://github.com/google-github-actions/release-please-action#action-inputs suggests there is a `target-branch` (also that it detects this automatically), but I don't see `default-branch` mentioned.